docs/docs/platform/database/index.mdx

```diff
@@ -148,12 +164,14 @@ nhost/seeds/default/002-products.sql
 
 ## Backups
 
-Databases on [Pro and Enterprise plans](https://nhost.io/pricing) are backed up automatically.
+Databases on the [Pro and Enterprise plans](https://nhost.io/pricing) are backed up automatically every day.
```

Comment:
Maybe worth mentioning the exact time of the backup here (GMT).

Reply:
We'll skip it for now as it's not 100% deterministic, whereas "every day" is.

Reply:
I wouldn't mention more than "daily backups". If you say it is at 02.00 UTC people will start having opinions about it while if you leave it out they will mostly not care much about it.